src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -149,19 +150,10 @@ class JesBackend extends Backend with LazyLogging {
   /**
    * Delete authentication file in gcs once workflow is in a terminal state.
    */
-  override def cleanUpForWorkflow(workflow: WorkflowDescriptor)(implicit ec: ExecutionContext): Future[Unit] = {
-    Future(JesConnection.storage.deleteObject(GoogleCloudStoragePath(gcsAuthFilePath(workflow))))
-  }
+  override def cleanUpForWorkflow(workflow: WorkflowDescriptor)(implicit ec: ExecutionContext): Future[Unit] = authenticated { connection =>
+    Future(connection.storage.deleteObject(GoogleCloudStoragePath(gcsAuthFilePath(workflow))))
+  } (workflow)
```

Comment:
if you `implicit` the `WorkflowDescriptor` as you did above, can you avoid the explicit `(workflow)`?